src/debugger/android/androidPlatform.ts

```diff
@@ -76,7 +83,7 @@ export class AndroidPlatform implements IAppPlatform {
     }
 
     private getPackageName(projectRoot: string): Q.Promise<string> {
-        return new Package(projectRoot).name().then(appName =>
+        return new Package(projectRoot, { fileSystem: this.fileSystem }).name().then(appName =>
```

Comment:
I like using the real Package class, so we can test more of the real code.
If we wanted to hide that knowledge from this class, we could start passing a serviceLocator parameter around instead of all the "dependencies", so this class would only know that the other classes uses services, but not which ones...